staging/src/k8s.io/sample-controller/hack/update-codegen.sh

```diff
@@ -19,7 +19,13 @@ set -o nounset
 set -o pipefail
 
 SCRIPT_ROOT=$(dirname "${BASH_SOURCE[0]}")/..
-CODEGEN_PKG=${CODEGEN_PKG:-$(cd "${SCRIPT_ROOT}"; ls -d -1 ./vendor/k8s.io/code-generator 2>/dev/null || echo ../code-generator)}
+
+if [[ $(go env GO111MODULE) == "on" ]]; then
```

Comment:
don't key off of explicit envvar values… this changes let go version. see suggestions at https://github.com/kubernetes/kubernetes/pull/85559#issuecomment-570234199 (which is a substantially similar PR that seems to have gone stale)

Reply:
Definition of CODEGEN_PKG moved into if-else condition